ltx-talk.dtx

```diff
@@ -81,8 +81,24 @@
   {A class for typesetting presentations}
 %    \end{macrocode}
 %
+% Get the right type of message.
+%    \begin{macrocode}
+\prop_gput:Nnn \g_msg_module_name_prop { talk } { ltx-talk }
+\prop_gput:Nnn \g_msg_module_type_prop { talk } { Class }
+%    \end{macrocode}
+%
 % Require the latest \LaTeX{} structures.
 %    \begin{macrocode}
+\IfFormatAtLeastF{2025-11-01}
+  {
+    \msg_new:nnnn { ltx-talk } { kernel-too-old }
+      { ltx-talk~ requires~ at~ least~ LaTeX2e~ 2025-11-01. }
+      {
+        You~ might~ need~ to~ compile~ using~ either~ lualatex-dev~ or~
+        pdflatex-dev.
```

Comment:
I will tidy up some spacing here after merging, and probably rephrase a bit.

Reply:
What's wrong with the spacing? I can fix that as well (and if you tell me your preferences I can abide by them in future code changes I do).